README.md

```diff
@@ -141,6 +141,8 @@ For local development, use the following for `.env`:
   ENVIRONMENT="development"
 ```
 
+Note: This is for new Macbook devices. We are currently using node version 14.7.0 in production. Before running `yarn dev`, please run this command `export NODE_OPTIONS=--openssl-legacy-provider`. You can save this in your `~/.zshrc` file as well. 
```

Comment:
I would write this is for people who use a newer version of node. Could also be people with other devices.

Reply:
Done